README.md

```diff
@@ -116,11 +116,15 @@ This allows you to hack on Caddy core (and optionally plug in extra modules at t
 
 ---
 
-You may embed directories into the Caddy executable:
+You may embed directories into the Caddy executable and serve them from the `embedded` filesystem:
 
 ```
 $ xcaddy build --embed foo:./sites/foo --embed bar:./sites/bar
 $ cat Caddyfile
+{
+    filesystem embedded embedded
```

Comment:
Before merging, I though it'd be better to clarify the stutter here. What do you think of this?
```suggestion
    # the second "embedded" an arbitrary name to be used with the `fs` directive
    filesystem embedded embedded
```

Reply:
hmmm. 

either this or `filesystem embedded myembedfs`

which clearly shows that it is arbitrary. 

which do you prefer?

Reply:
Yes, I like avoiding the stutter. Unless I'm mistaken, it's the first of the two arguments that's the arbitrary name, though, right? cf. the spec and examples at https://caddyserver.com/docs/caddyfile/options#file-systems

Reply:
yeah its the first lol i was wrong

Reply:
Yeah, you're right. Pick either approach you like, as long as it's obvious to the user.

Reply:
I combined both of your suggestions. Let me know if it seems too verbose and I can pare it back a bit 🫣

Reply:
Love it 👍🏽